docs/Guides/Automate/WebhookAction.md

```diff
@@ -79,6 +79,15 @@ The webhook may be resent automatically for a few reasons:
 
 If you require more than 60 seconds to process the webhook you should start a background workflow to do the processing. (We are fans of the [Temporal](https://www.temporal.io/) workflow engine.)
 
+If your webhook returns an error status code and you would like to display information about this error to the end user in the Vendasta platform you can include a JSON encoded response body that includes the key `human_readable` with a message to be shown to the user
```

Comment:
Does "error status code" equal just 4XX or also 5XX? I think that it is both but the 5xx errors are only displayed once the maximum number of retries have been reached. We might want to clarify that in the description.